openedx/core/lib/block_structure/transformers.py

```diff
@@ -81,8 +86,8 @@ def transform(self, block_structure):
         The given block structure is transformed by each transformer in the
         collection, in the order that the transformers were added.
         """
-        for transformer in self._transformers:
-            transformer.transform(self.usage_info, block_structure)
+        self._transform_optimized(self._transformers['optimized'], block_structure)
+        self._transform_nonoptimized(self._transformers['nonoptimized'], block_structure)
```

Comment:
No longer need to pass in the first parameters to the above 2 calls as they are instance members.